packages/scripts/build-wasm.sh

```
@@ -103,9 +103,16 @@ if [ "$USE_WASM_PACK" = true ]; then
     # Build using wasm-pack
     echo "Building with wasm-pack..."
     
+    # Set build-time environment variables
+    export GIT_COMMIT=$(git rev-parse --short HEAD 2>/dev/null || echo "unknown")
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)